auction/src/services/SessionService.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import Cookies from 'js-cookie';
+
+function setAuthorizationHeaderDefault(token) {
+    axios.defaults.headers.common['Authorization'] = `Bearer ${token}`;
+}
+
+export const login = (dispatch, loginData) => {
+    axios.post('http://localhost:9000/login', loginData)
```

Comment:
move `http://localhost:9000` to constant which you will export from another js file. Use import to get that constant.

Reply:
Done